src/test/java/com/google/cloud/spanner/jdbc/JdbcArrayTest.java

```diff
@@ -35,42 +41,182 @@ public void testCreateArrayTypeName() throws SQLException {
     // Note that JDBC array indices start at 1.
     JdbcArray array;
     array = JdbcArray.createArray("BOOL", new Boolean[] {true, false, true});
-    assertEquals(array.getBaseType(), Types.BOOLEAN);
-    assertEquals(((Boolean[]) array.getArray(1, 1))[0], Boolean.TRUE);
+    assertThat(array.getBaseType()).isEqualTo(Types.BOOLEAN);
+    assertThat(((Boolean[]) array.getArray(1, 1))[0]).isEqualTo(Boolean.TRUE);
+    try (ResultSet rs = array.getResultSet()) {
+      assertThat(rs.next()).isTrue();
```

Comment:
why not just assertEquals and assertTrue?

Reply:
This library (and also the Java Spanner client) mostly use [Truth](https://truth.dev/) for assertions, but some of the older classes use JUnit assertions. To reduce the mixing of the two, I tend to change the older classes to Truth when (larger) changes are made to them.